packages/experimental/labs-functions/src/functions/github/handler.ts

```diff
@@ -2,6 +2,71 @@
 // Copyright 2023 DXOS.org
 //
 
-import { type FunctionHandler } from '@dxos/functions';
+import { Octokit, type RestEndpointMethodTypes } from '@octokit/rest';
 
-export const handler: FunctionHandler<any> = async ({ event, context }) => {};
+import { scheduleTask } from '@dxos/async';
+import { Context } from '@dxos/context';
+import { type Schema, TypedObject } from '@dxos/echo-schema';
+import { type FunctionSubscriptionEvent, type FunctionHandler } from '@dxos/functions';
+import { invariant } from '@dxos/invariant';
+import { PublicKey } from '@dxos/keys';
+import { log } from '@dxos/log';
+
+type GithubContributors = RestEndpointMethodTypes['repos']['listContributors']['response']['data'];
+type GithubUser = RestEndpointMethodTypes['users']['getByUsername']['response']['data'];
+
+export const handler: FunctionHandler<FunctionSubscriptionEvent> = async ({ event, context }) => {
+  const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+
+  scheduleTask(new Context(), async () => {
+    for (const objectId of event.objects) {
+      const project = context.client.spaces.query({ id: objectId }).objects[0];
```

Comment:
I think this is clearer:
```
const { objects: projects } = client.query();
const [ project ] = projects[0];
```
But perhaps we need a `getObject(id: string)` API since this comes up a lot.